sphinx/source/start_network.rst

```diff
@@ -95,6 +95,34 @@ Other members are then allowed to vote for the proposal, using the proposal ID r
 
 Once a quorum of members have approved the network opening (``"result":true``), the network is opened to users (see :ref:`Example App` for a simple business logic and :term:`JSON-RPC` transactions). It is only then that users are able to execute transactions on the business logic defined by the enclave file (``--enclave-file`` option to ``cchost``).
 
+Add new nodes to an open network
+--------------------------------
+
+Once the network has been opened by members, it is possible to add new nodes to the network (e.g. to replace a retired node or add a new version of the code). The new node should be started with the ``join`` option:
+
+.. code-block:: bash
+
+    $ cchost --enclave-file /path/to/enclave_library --enclave-type debug
```

Comment:
I wonder if we should format this with one option per line for readability? It looks a bit like a wall of text otherwise.